image.py

```diff
@@ -40,7 +37,58 @@ def enhance_image_clahe(self,image,limit,demo=1,invert=0):
             cv2.waitKey()
         return enhanced
 
-    def cropper(self, test_img, demonstration = True):
```

Comment:
Return size can be parametrized. The default value can be (299,299).
Thanks for your contribution.

Reply:
Applied requested changes